project/api/tests.py

```diff
@@ -53,13 +55,102 @@ def setUpTestData(self):
                                            aliases=[],
                                            links=[])
         new_nation.save()
+
         child_nation = Nation.objects.create(name="Test Child Nation",
-                                           url_id="test_child_nation",
-                                           color="ccc",
+                                             url_id="test_child_nation",
+                                             color="ccc",
+                                             references=[
+                                                 "https://en.wikipedia.org/wiki/Test"],
+                                             aliases=[],
+                                             links=[])
+        child_nation.save()
+        Territory.objects.create(start_date="1444-11-11",
+                                 end_date="2018-01-01",
```

Comment:
I have a small proposal: Right now it's very hard to know which territories come first, last, which overlap, i have dates in 1444 and some random months of 2018.

We could just create territories in the format `00xx-01-01`, where xx is 00-99. And the date overlapping tests would be easy to read.

cc @whirish what do you think?

Reply:
Love this idea :+1: